rke/templates/templates.go

```diff
@@ -96,16 +97,19 @@ var TemplateIntroducedRanges = map[string][]string{
 func LoadK8sVersionedTemplates() map[string]map[string]string {
 	return map[string]map[string]string{
 		kdm.Calico: {
-			">=1.22.0-rancher1-1":                    calicov3211,
-			">=1.21.0-rancher1-1 <1.22.0-rancher1-1": calicov319,
-			">=1.20.4-rancher1-1 <1.21.0-rancher1-1": calicov3171,
-			">=1.19.4-rancher1-2 <1.20.4-rancher1-1": calicov3165,
-			">=1.19.0-rancher0 <1.19.4-rancher1-2":   calicov3160,
-			">=1.17.4-rancher0 <1.19.0-rancher0":     calicov117Privileged,
-			">=1.17.0-rancher0 <1.17.4-rancher0":     calicov117,
-			">=1.16.8-rancher0 <1.17.0-rancher0":     calicov117Privileged,
-			">=1.16.4-rancher1 <1.16.8-rancher0":     calicov117,
-			">=1.16.0-alpha <1.16.4-rancher1":        calicov116,
+			">=1.24.9-rancher1-1 < 1.25.2-rancher1-1":  calicov3225,
+			">=1.23.15-rancher1-1 < 1.24.2-rancher1-1": calicov3225,
+			">=1.24.2-rancher1-1 <1.24.9-rancher1-1":   calicov3211,
+			">=1.22.0-rancher1-1 <1.23.15-rancher1-1":  calicov3211,
```

Comment:
We will need to break this range to accommodate 1.22.17 which will (unexpectedly) be released with Dec 2022 k8s patches and will need to use `calicov3225` template.

Reply:
done